### PR TITLE
Working on email decorators

### PR DESCRIPTION
**Pull request by banurekha**

### CODE REVIEW

**Comment by banurekha** [comment]
These values are used in email at the moment and I had preferred them in decorator since there is no styling needed at email.

File: app/decorators/sipity/decorators/email_notification_decorator.rb [hunk 1]
```diff
@@ -42,10 +36,21 @@ def work_show_path
         view_context.work_path(entity.id)
```
Comment (jeremyf) [inline app/decorators/sipity/decorators/email_notification_decorator.rb]: We can't use `work_path`; That resolves to `/works/:id`. We need the URL, so we have `http://sipity-staging.library.nd.edu/works/:id`

I believe we will need a configuration option.

**Comment by jeremyf** [comment]
@banurekha Sounds reasonable. Keep the `to_sentence` usage.

**Comment by jeremyf** [comment]
I'm rebasing and going to merge. Good stuff!

**Pull request closed without merging**